tests/functional_test.js

```diff
@@ -222,8 +214,7 @@ describe('functional', function () {
                 return;
             }
 
-            var domain = helpers.domainCheck('https://maxcdn.bootstrapcdn.com/');
-            var uri = domain + root + '/' + name;
+            var uri = 'https://maxcdn.bootstrapcdn.com/' + root + '/' + name;
```

Comment:
I'm on the fence, but I think I want to keep it. This is how I validate uploads to S3 before purging cache.